Environmental/new_crawlers/Weather forecast/cork_weather_forecast/producer.py

```diff
@@ -0,0 +1,48 @@
+import os
+import json
+import requests
+import datetime
+from kafka import KafkaProducer
+from dotenv import load_dotenv
+
+CORK_LAT = 51.804819
+CORK_LON = -8.301993
+THESS_RESOLUTION = "18km"
+FROM_DATE = datetime.datetime.now().strftime("%Y-%m-%d")
+TO_DATE = (datetime.datetime.now() + datetime.timedelta(days=7)).strftime("%Y-%m-%d")  # TO_DATE = FROM_DATE + 7 days
+VARIABLES = ["temperature2m", "rh2m", "windspeed10m", "precipitation"]
+TIMEZONE = "Europe/Athens"
+
+
+def get_seven_day_forecast():
+    params = {
+        "lat": CORK_LAT,
+        "lon": CORK_LON,
+        "resolution": THESS_RESOLUTION,
+        "from_date": FROM_DATE,
+        "to_date": TO_DATE,
+        "variables": ','.join(VARIABLES),
+        # "timezone": TIMEZONE,
+        "apikey": os.environ["API_KEY"]
+    }
+    url = os.environ["DRAXIS_API"] + os.environ["FORECAST_URI"]
+    response = requests.get(url=url, params=params)
+    return response.json()
+
+
+def broadcast_to_topic(topic, forecast):
+    producer.send(topic, forecast)
+
+
+load_dotenv()
```

Comment:
we should add an `.env` file in the repo with some default or empty values, so that we know what we need to change